systems/common/notification/notification.go

```diff
@@ -35,11 +35,15 @@ func (l NotificationScope) Value() (driver.Value, error) {
 type NotificationType uint8
 
 const (
-	TYPE_INAVLID  NotificationType = 0
-	TYPE_INFO     NotificationType = 1
-	TYPE_WARNING  NotificationType = 2
-	TYPE_ERROR    NotificationType = 3
-	TYPE_CRITICAL NotificationType = 4
+	TYPE_INAVLID             NotificationType = 0
```

Comment:
There is a typo here it should be `INVALID`.

Reply:
addressed